libcxx/include/__memory/allocator_traits.h

```diff
@@ -319,11 +315,7 @@ struct _LIBCPP_TEMPLATE_VIS allocator_traits
         __enable_if_t<!__has_destroy<allocator_type, _Tp*>::value> >
     _LIBCPP_INLINE_VISIBILITY _LIBCPP_CONSTEXPR_SINCE_CXX20
     static void destroy(allocator_type&, _Tp* __p) {
-#if _LIBCPP_STD_VER >= 20
```

Comment:
Why is using `destroy_at` better than simply invoking the destructor? One costs a lot less to instantiate. 
It's  not a customization point, right?